requirements-dev.txt

```diff
@@ -7,6 +7,7 @@
 -r tools/c7n_kube/requirements.txt
 -r tools/c7n_mailer/requirements.txt
 -r tools/c7n_org/requirements.txt
+-r tools/dev/requirements.txt
```

Comment:
hmm.. not sure we want this yet as part of req-dev which is used in ci, we've run into various issues on pygit2 compilation using latest because it needs more recent versions of libgit2 then what's commonly in distros.

Reply:
OK, removed it. I've also added a `pytest.importorskip('pygit2')` to the test that comes with this PR.